app/routes/wow.ilvl-export-search.tsx

```
@@ -312,15 +322,25 @@ const IlvlExportSearchComponent = () => {
         />
         <InputWithLabel
           labelTitle="Ranking"
           name="rankingWP"
           type="number"
           value={formValues.rankingWP.toString()}
           min={1}
-          onChange={(e) => setFormValues(prev => ({ ...prev, rankingWP: parseInt(e.target.value) }))}
+          onChange={(e) =>
+            setFormValues((prev) => ({
+              ...prev,
+              rankingWP: parseInt(e.target.value)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)